source/_patterns/01-molecules/navigation/site-header-nav-bar~secondary.json

```diff
@@ -4,35 +4,24 @@
 
   "linkedItems": [
     {
-      "text": "Newsletters",
-      "path": "#",
-      "classes": "nav-secondary__item"
+      "text": "Search",
+      "rel": "search",
+      "classes": "nav-secondary__item nav-secondary__item--search",
+      "path": "#"
     },
 
     {
-      "text": "About<span class=\"visuallyhidden\"> eLife</span>",
-      "path": "#",
-      "classes": "nav-secondary__item"
+      "text": "Alerts",
+      "classes": "nav-secondary__item nav-secondary__item--alert",
```

Comment:
Another followup ticket to address how we can add a custom class to a nav item. Current solution is to check for the word alert in the text.